subsys/bluetooth/controller/Kconfig

```diff
@@ -525,7 +531,7 @@ config BT_CTLR_ADV_PERIODIC
 	  Controller.
 
 config BT_CTLR_ADV_PERIODIC
-	bool "LE Periodic Advertising in Advertising State [EXPERIMENTAL]" if BT_LL_SW_SPLIT
+	bool "LE Periodic Advertising in Advertising State (Split Link Layer) [EXPERIMENTAL]" if BT_LL_SW_SPLIT
```

Comment:
comment not directly related to this line, but cannot comment below.

Why was this removed in last update ?
https://github.com/zephyrproject-rtos/zephyr/blob/65a659e7f7e2ad76a2406f8d6db57bca99ee25b3/subsys/bluetooth/controller/Kconfig#L537-L538

Only the help text was supposed to be remove if you read this comment https://github.com/zephyrproject-rtos/zephyr/pull/42319#discussion_r811024770.

This code part still suffers the double prompt issue, as there are two `config BT_CTLR_ADV_PERIODIC` after one another, and then a `depends on BT_CTLR_ADV_PERIODIC` here:
https://github.com/zephyrproject-rtos/zephyr/blob/e4e8b78f93a7871c4494bf6b2e5596e97fcf7c12/subsys/bluetooth/controller/Kconfig#L524-L539

Especially pay attention to this line:
https://github.com/zephyrproject-rtos/zephyr/blob/e4e8b78f93a7871c4494bf6b2e5596e97fcf7c12/subsys/bluetooth/controller/Kconfig#L539

Show you still need
```
config PROMPT_ADJUSTMENT_WORKAROUND
	bool
```
just no help text.